SUASSystem/SUASSystem/settings.py

```diff
@@ -3,8 +3,8 @@ class GCSSettings(object):
     UAV_CONNECTION_STRING = "tcp:127.0.0.1:14551"
 
     INTEROP_URL = "http://10.10.130.2:8000"
-    INTEROP_USERNAME = "Flint"
-    INTEROP_PASSWORD = "271824758"
+    INTEROP_USERNAME = "img_proc_test"
```

Comment:
Please comment out the other values, do not remove since we need both sets for testing.